### PR TITLE
Verbal autopsy

### PR DESCRIPTION
**Pull request by katembu**
DO NOT MERGE

### CODE REVIEW

**Comment by katembu** [comment]
@biyeun you can review and merge this report. It's just a replica of existing MVIS Report.  I have done   necessary mapping on MVP-Sauri domain. Other domains will be deployed later.

File: custom/_legacy/mvp/__init__.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
-from mvp.reports import mvis, chw
+from mvp.reports import mvis, chw, va
 
 CUSTOM_REPORTS = (
     ('Custom Reports', (
         mvis.HealthCoordinatorReport,
-        chw.CHWManagerReport
+        chw.CHWManagerReport,
+        va.VerbalAutopsyReport
```
Comment (biyeun) [inline custom/_legacy/mvp/__init__.py]: pep8, missing comma at the end

**Comment by katembu** [comment]
@biyeun

**Comment by biyeun** [comment]
@katembu I'm trying to do a couple of system optimizations on our end, as the new couch application that this report creates could introduce a serious toll on our system. I will update you on Monday in regards to what I've discovered about the situation...

**Comment by katembu** [comment]
@biyeun have you been able to get break through with the optimization? let me know if its not possible to merge the new Report soon, I can cherry-pick some minor changes for previous report included in this pull request and make a different pull request.

**Comment by biyeun** [comment]
Yes, it's better you cherry pick those changes. I will try to aim to merge
the rest in middle of next week...

**Comment by katembu** [comment]
Closing this in favour of #5386 which follow new structure. @biyeun

**Pull request closed without merging**